src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -4,40 +4,38 @@
 import edu.kis.vh.nursery.collection.IntLinkedList;
 
 public class DefaultCountingOutRhymer {
-	IntArrayStack stack = new IntArrayStack();
-	IntLinkedList list = new IntLinkedList();
+	IntStack stack = null;
 
 	public DefaultCountingOutRhymer(IntArrayStack stack) {
 		super();
 		this.stack = stack;
 	}
-	public DefaultCountingOutRhymer() {
+	public DefaultCountingOutRhymer(IntLinkedList list) {

```

Comment:
One constructor: public DefaultCountingOutRhymer(IntStack stack) would be enough